### PR TITLE
added readme description

### PR DESCRIPTION
**Pull request by marisaaquilina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# jupyterlab-zen-mode
\ No newline at end of file
+# JupyterLab Clarity Mode
```
Comment (markellekelly) [inline README.md]: Why did you remove the hyphens?

Comment (javag97) [reply README.md]: Very efficient use of PR team

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
+# Clarity mode is a document-focused, decluttered mode of JupyterLab. It reveals relevant notebook tools in response to different user activities.
```
Comment (markellekelly) [inline README.md]: Is it a mode of JupyterLab or an alternative to it?

File: README.md (hunk 1 above)
Comment (marisaaquilina) [reply README.md]: @markellekelly The hyphens originate from the repo name but I've changed it to a h1.
